src/components/Tree/DataZTree/components/ZTree/index.vue

```diff
@@ -439,21 +440,21 @@ div.rMenu li {
 }
 
 .menu-item {
-    font-size: 12px;
-    padding: 0 16px;
-    position: relative;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    color: #606266;
-    height: 24px;
-    line-height: 24px;
-    box-sizing: border-box;
-    cursor: pointer;
+  font-size: 12px;
+  padding: 0 16px;
+  position: relative;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: #606266;
+  height: 24px;
+  line-height: 24px;
+  box-sizing: border-box;
+  cursor: pointer;
 
-    i {
-      width: 15px;
-    }
+  i {
+    width: 15px;
+  }
 }
 
 .dropdown-menu {
```

Comment:
I'm sorry but I do not have your original code to analyze it. Please share the code so that I can review it for you.